routes/tags.php

```diff
@@ -6,15 +6,13 @@
     'middleware' => ['scope:tags']
 ], function () use ($router) {
     // Public access
-    $router->get('/', 'TagsController@index');
-    $router->get('/{id}', 'TagsController@show');
+    resource($router, '/', 'TagsController', [
```

Comment:
This pattern happens in several places - there should clearly be either an `public-read` option or second helper for this.